.github/workflows/deploy-to-dev.yml

```
@@ -42,32 +42,15 @@ jobs:
           docker build -t ${{ secrets.DOCKER_USERNAME }}/${{ secrets.DOCKER_REPO }} .
           docker push ${{ secrets.DOCKER_USERNAME }}/${{ secrets.DOCKER_REPO }}
 
```

환경 변수 복사해서 넘기는 방법도 추가해본다면 좋을 것 같습니다!

```suggestion
- name: Send To Env
  uses: appleboy/scp-action@master
  with:
    username: ec2-user
    host: ${{ secrets.EC2_HOST }}
    key: ${{ secrets.EC2_PRIVATE_KEY }}
    source: "./.env"
    target: "/home/ec2-user"
```